src/tasks/make-util.js

```diff
@@ -0,0 +1,1632 @@
+var check = require('validator');
```

Comment:
>va [](start = 0, length = 2)

Where did this file come from? It's... really big, so I was hoping it was just pulled from somewhere else so I don't have to review it. :-)